scissors/scissors_game/scissors.h

```diff
@@ -7,6 +7,14 @@ static enum {
 	PAPER,
 	SCISSORS
 } element_id;
+
+static enum {
+	DRAW,
+	LOST,
+	WINN
+} game_results;
```

Comment:
What's the purpose of variable `game_results`? - Most probably you've meant something different.